webapp/src/components/icons.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export const ChannelHeaderButtonIcon = () => (
+    <i
+        className='icon fa fa-plug'
+        style={{fontSize: '15px', position: 'relative', top: '-1px'}}
+    />
+);
```

Comment:
Can you add the missing newlines on all new files in this PR? Is there a way you can configure your editor to automatically add these on save?